evs.mk

```diff
@@ -10,9 +6,11 @@ PRODUCT_PACKAGES += \
     android.hardware.automotive.evs@1.1.xt \
     android.automotive.evs.manager@1.1 \
     evs_app_xt \
-    android.hardware.automotive.evs@1.1-xt
+    android.hardware.automotive.evs@1.1-xt \
```

Comment:
Hi @DKlimenko, do you really need this change? It is not functional. If really needed, please, move it to a separate commit.